Dashboard.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from 'react';
+import { getStudents } from './api';
+
+function Dashboard() {
+  const [students, setStudents] = useState([]);
+
+  useEffect(() => {
+    getStudents().then(setStudents);
```

Comment:
Consider handling errors that may occur during the API call to get students. This can be done by using a try-catch block or a .catch() method if the API call returns a promise.